lib/mix/tasks/elixir_make.precompile.ex

```diff
@@ -47,7 +47,11 @@ defmodule Mix.Tasks.ElixirMake.Precompile do
       Mix.Project.build_structure()
     after
       app_priv = Path.join(Mix.Project.app_path(config), "priv")
-      File.rm!(app_priv)
+
+      for include <- paths,
+          file <- Path.wildcard(Path.join(app_priv, include)) do
+        File.rm_rf!(file)
```

Comment:
```suggestion
        File.rm_rf(file)
```

Reply:
That's because you can have directories in there, and once you delete a directory, you will also delete its entries?

Reply:
Oh that's true. But when I tested `File.rm_rf!` locally, it didn't raise any error even if I passed a non-exists path to it...